ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/RejectionReason.scala

```diff
@@ -50,6 +50,18 @@ object RejectionReason {
       "The maximum record time of the command exceeded"
   }
 
+  /** The ledger time of the transaction submission violated one of the
```

Comment:
To keep maintenance easier, please align the order of case classes, case statements, etc. to the order in the proto definition.

Reply:
Moved the new definitions everywhere to the end.

Reply:
Thanks!